scss/components/_panel-secondary.scss

```diff
@@ -0,0 +1,59 @@
+.panel-secondary {
+  $panel-control-margin: 10px;
+
+  box-shadow: $box-shadow-block;
+
+  .panel-body {
+    padding: $case-panel-secondary-padding;
+  }
+
+  .panel-footer {
+    padding: $case-panel-secondary-padding-small $case-panel-secondary-padding;
+  }
+
+  .panel-heading {
+    background: $crm-white;
+    line-height: 1;
+    padding: $case-panel-secondary-padding;
+    position: relative;
+
+    &::after {
+      $border-margin-right: $case-panel-secondary-padding + ($case-panel-secondary-padding - $case-panel-secondary-padding-small);
+
+      border-bottom: 1px solid $gray-light;
+      bottom: 0;
+      content: '';
+      display: block;
+      height: 0;
+      left: $case-panel-secondary-padding-small;
+      position: absolute;
+      width: calc(100% - #{$border-margin-right});
+    }
+  }
+
+  // Vertically aligns a control (select, btn-group, link)
+  // inside the heading
+  .panel-heading-control {
+    display: block;
+    margin-left: 0;
+    margin-top: -13px;
+    position: relative;
+    top: 6px;
+
+    + & {
+      margin-left: $panel-control-margin;
+    }
+  }
+
+  a.panel-heading-control { /* stylelint-disable-line selector-no-qualifying-type */
+    line-height: 30px;
```

Comment:
isn't this [$crm-control-height](https://github.com/civicrm/org.civicrm.shoreditch/blob/b0546e66dda5b1914838ca70443681a6c7c62cc6/scss/civicrm/_variables.scss#L4)?

Reply:
That's only for the "legacy mode" of shoreditch (it's in the _civicrm/_ folder), but i used `$input-height-base` which is the bootstrap equivalent, nice catch